src/commands/openapi/resolve.ts

```diff
@@ -139,14 +136,13 @@ export default class OpenAPIResolveCommand extends BaseCommand<typeof OpenAPIRes
     /** List of circular reference paths. */
     circularRefs: string[],
   ): void {
-    // eslint-disable-next-line @typescript-eslint/no-this-alias
     const self = this;
     const createdRefs = new Set<string>();
 
     function replaceRef(schemaName: string, propertyName: string, refSchemaName: string) {
       const schema = schemas[schemaName];
-      if ('properties' in schema) {
-        schema.properties![propertyName] = { $ref: `#/components/schemas/${refSchemaName}` };
```

Comment:
Rewrote this off of the non-null assertion because that was being picked up by their default ruleset.

![Screenshot 2025-07-01 at 6 04 00 PM](https://github.com/user-attachments/assets/f9f36bd1-b82d-4c8c-9a85-0a08991aab82)